### DISCUSSION TITLE
use pygment to stylize the docs code-blocks

### DISCUSSION THREAD

**Discussion by 2bndy5** [Textualize/rich#1140] [Ideas] [answered]
I noticed an inconsistency in 1 of the doc's code-blocks under [Progress Display->Table Columns](https://rich.readthedocs.io/en/latest/progress.html#table-columns).

Actually, the first thing that caught my eye was a missing backtick ( \` )  in the rst reference for the `Tables` class (missing before the tilda `~`).

But then I noticed all the code blocks are written like quoted blocks (just an extra indent without the<br> `.. code-block:: python` directive). Since this lib emphasizes style, why not add some flare to the code-blocks in the docs. You can even specify your own pygment style to use in the _conf.py_ (I [monkey-patched a "dark-plus" style](https://github.com/2bndy5/CircuitPython_nRF24L01/blob/dd405038aebd74a42862620dccbcbc23770f46d5/docs/conf.py#L113-L176) into my docs - [see example](https://circuitpython-nrf24l01.readthedocs.io/en/latest/examples.html#id6))

p.s. there's also a [nifty Sphinx extension that adds a "copy" button](https://sphinx-copybutton.readthedocs.io/en/latest/) to all code-blocks in the docs

**Comment by willmcgugan** [comment, marked as answer]
Fixed the table columns section.

All the code blocks are syntax highlighted that I can see.

I like that copy button extension, I might use that.
